packages/palette/package.json

```diff
@@ -20,7 +20,7 @@
     "prepublishOnly": "yarn clean && yarn compile && yarn type-declarations",
     "prettier-project": "yarn prettier --write 'src/**/*.{ts,tsx}'",
     "release": "auto shipit && node scripts/clean-up-after-auto.js",
-    "test": "yarn type-check && yarn jest",
+    "test": "yarn type-check && yarn jest --runInBand",
```

Comment:
Should this stay here or is it a temporary fix? I'm not too confident yet with the inner workings of palette.

Reply:
I think this should stay unless we can pass args thru the CircleCI config. Either way it's fine to merge. Coming from https://jestjs.io/docs/en/troubleshooting#tests-are-extremely-slow-on-docker-andor-continuous-integration-ci-server